### ISSUE TITLE
[BUG]  Changelog is not updated to 11.3.1

### ISSUE THREAD

**Issue by uniuuu** [docker-mailserver/docker-mailserver#3120]
### Miscellaneous first checks

- [X] I checked that all ports are open and not blocked by my ISP / hosting provider.
- [X] I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.

### Affected Component(s)

No debug

### What happened and when does this occur?

Server keeps sending notification:

```
Subject: Mailserver update available! [ 11.3.0 --> 11.3.1 ]

Hello ****!

There is a docker-mailserver update available on your host: localhost

Current version: 11.3.0
Latest  version: 11.3.1

Changelog: https://github.com/docker-mailserver/docker-mailserver/blob/master/CHANGELOG.md
```

In the commit b75fc448eae0185b991d8707764fc7f4b5381121 which 11.3.1 release tag the CHANGELOG.md as well is on 11.3.0 instead 11.3.1
Refer: https://github.com/docker-mailserver/docker-mailserver/tree/b75fc448eae0185b991d8707764fc7f4b5381121  

### What did you expect to happen?

updated to  11.3.1

### How do we replicate the issue?

1. Create container based on 11.3.1 image  `docker.io/mailserver/docker-mailserver:latest`

https://hub.docker.com/layers/mailserver/docker-mailserver/11.3.1/images/sha256-e15b8840db6fbe3a95052b2ab903e44a3c0c167fcf142ea83fd6f9dfa13f1447?context=explore

### DMS version

v11.3.1

### What operating system is DMS running on?

Linux

### Which operating system version?

AlmaLinux 8.7

### What instruction set architecture is DMS running on?

AMD64 / x86_64

### What container orchestration tool are you using?

Docker

### docker-compose.yml

```yml
services:
  mailserver:
    image: docker.io/mailserver/docker-mailserver:latest
    container_name: mailserver
    # If the FQDN for your mail-server is only two labels (eg: example.com),
    # you can assign this entirely to `hostname` and remove `domainname`.
    hostname: mail
    domainname: example.com
    env_file: mailserver.env
    # More information about the mail-server ports:
    # https://docker-mailserver.github.io/docker-mailserver/edge/config/security/understanding-the-ports/
    # To avoid conflicts with yaml base-60 float, DO NOT remove the quotation marks.
    ports:
      - "25:25"    # SMTP  (explicit TLS => STARTTLS)
      - "143:143"  # IMAP4 (explicit TLS => STARTTLS)
      - "465:465"  # ESMTP (implicit TLS)
      - "587:587"  # ESMTP (explicit TLS => STARTTLS)
      - "993:993"  # IMAP4 (implicit TLS)
    volumes:
      - ./docker-data/dms/mail-data/:/var/mail/
      - ./docker-data/dms/mail-state/:/var/mail-state/
      - ./docker-data/dms/mail-logs/:/var/log/mail/
      - ./docker-data/dms/config/:/tmp/docker-mailserver/
      - /etc/localtime:/etc/localtime:ro
    restart: always
    stop_grace_period: 1m
    cap_add:
      - NET_ADMIN
    healthcheck:
      test: "ss --listening --tcp | grep -P 'LISTEN.+:smtp' || exit 1"
      timeout: 3s
      retries: 0
```

### Relevant log output

```Text
Subject: Mailserver update available! [ 11.3.0 --> 11.3.1 ]

Hello ****!

There is a docker-mailserver update available on your host: localhost

Current version: 11.3.0
Latest  version: 11.3.1

Changelog: https://github.com/docker-mailserver/docker-mailserver/blob/master/CHANGELOG.md
```
```


### Other relevant information

_No response_

### What level of experience do you have with Docker and mail servers?

- [ ] I am inexperienced with docker
- [X] I am rather experienced with docker
- [ ] I am inexperienced with mail servers
- [X] I am rather experienced with mail servers
- [ ] I am uncomfortable with the CLI
- [X] I am rather comfortable with the CLI

### Code of conduct

- [X] I have read this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree
- [X] I have read the [README](https://github.com/docker-mailserver/docker-mailserver/blob/master/README.md) and the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/) and I searched the [issue tracker](https://github.com/docker-mailserver/docker-mailserver/issues?q=is%3Aissue) but could not find a solution

### Improvements to this form?

TLTR

**Comment by polarathene** [comment]
Did you miss the pinned issue?: https://github.com/docker-mailserver/docker-mailserver/issues/2952#issuecomment-1364365934

The release is late unfortunately, but workarounds have been suggested in the pinned issue I linked. You can otherwise ignore the update notice and subscribe to the pinned issue, it will be closed when a new release is published.

**Comment by georglauterbach** [comment]
Duplicate of #2952.
